AirCasting/CreateSessionViews/TurnOnBluetoothView.swift

```diff
@@ -11,6 +11,7 @@ import SwiftUI
 
 struct TurnOnBluetoothView: View {
     @State private var isPowerABLinkActive = false
+    @State private var presentRestartNextScreen = false
```

Comment:
I think the `Next` part here is not needed and `presentNextScreen` would be clearer